fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.json

```diff
@@ -100,7 +99,8 @@
    "fieldname": "email",
    "fieldtype": "Data",
    "label": "Email",
-   "options": "Email"
+   "options": "Email",
+   "read_only": 1
```

Comment:
so this makes profiles read-only?

Reply:
Not this field, but all the permissions listed below do